operate/schema/src/main/java/io/camunda/operate/schema/IndexTemplateDescriptorsConfigurator.java

```diff
@@ -129,7 +130,15 @@ public SequenceFlowTemplate getSequenceFlowTemplate(
   }
 
   @Bean
-  public TaskTemplate getUserTaskTemplate(final DatabaseInfo databaseInfo) {
+  public UserTaskTemplate getUserTaskTemplate(
```

Comment:
❔ Why do we still need it? Because of importer? Should we create a follow-up task to fully remove it?

Reply:
@sdorokhova Removed UserTaskTemplate and importers for UserTasks in the followup commit